test/e2e/.eslintrc.cjs

```diff
@@ -3,4 +3,7 @@ const synpressPath = path.join(process.cwd(), '/node_modules/@agoric/synpress');
 
 module.exports = {
   extends: `${synpressPath}/.eslintrc.js`,
+  rules: {
+    'cypress/unsafe-to-chain-command': 'off',
```

Comment:
why is this necessary? what are consequences?

please justify in a comment

Reply:
If I keep this rule on, I get this error with some commands that are chained:

```
error    It is unsafe to chain further commands that rely on the subject after this command. It is best to split the chain, chaining again from `cy.` in a next command line  cypress/unsafe-to-chain-command
```

Specifically, it happens with the tests where I use `checkFieldValue` custom command:

```
    cy.checkFieldValue(fieldName, expectedValue).then(isMatched => {
      expect(isMatched).to.be.true;
    });
```

The rule can be helpful in certain scanerios, but when it comes to custom commands, it can feel limiting. I've addressed all potential scenarios within my custom command, so encountering a null or undefined value isn't an issue in this context. It simply signifies that the custom command encountered an error.

Reply:
I'm reluctant to approve a lint suppression, especially in a tool I haven't used myself.

Could you accomplish this another way? Perhaps by making the expectation within the `checkFieldValue`? By the name it sounds like it would do the `expect` itself.

Reply:
Oh, I see you do have it throwing now. So why ever return `true`? Just remove the return value and stop reading it here. Consider also renaming for clarity

Reply:
Yeah, it makes it simpler. Updated